specification/cognitiveservices/Language.Conversations/tsp-output/@azure-tools/typespec-autorest/azure.ai.language.conversations.json

```diff
@@ -323,6 +323,55 @@
         "modelAsString": true
       }
     },
+    "Azure.Core.Foundations.Error": {
```

Comment:
Hmm. This still comes out different. @tg-msft @johanste, to match the old swagger example, should he just copy / redefine the error response and friends so that the swagger is exactly the same? This seems congruent, but I wonder if the different name would mess up DPG. Or maybe we could fix that with a client tsp if necessary. What do you think?

Reply:
+@m-nash for our options to tweak this but still have the generator pick it up as the common error model

Reply:
For now we'll go with this. It's generally correct, and if we ever need to copy in the short term it won't be much work.